master/getting-started/kubernetes/installation/manifests/app-layer-policy/etcd/calico-networking/calico-node.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 layout: null
 ---
-{% helm %}
+{% helm calico-node.yaml %}
```

Comment:
do we want to use a single argument for the specific template?  I'm suspicious we'll want to do more helm customization in the future, so would it be better to do

`{% helm template=calico-node.yaml %}`

or similar...